### ISSUE TITLE
Nose-Hoover thermostat

### ISSUE THREAD

**Issue by slaymuel** [TeamOverkill/md#25]
Maybe start with only one and then extend to use several chains

**Comment by SHervoe-Hansen** [comment]
I think this is a little over do atm, I think we should rather focus on trying to run the lab. Nose-hover can be implemented for next year.

**Comment by SHervoe-Hansen** [comment]
But I am willing to look at it when I return home as we talked about! 👍

**Comment by slaymuel** [comment]
I agree that it should not be the focus for the md-course work. I'm just eager to use the code for real research :P
